src/standalone/Snow/Snow.jl

```diff
@@ -24,16 +61,282 @@ Base.@kwdef struct SnowParameters{FT <: AbstractFloat, PSE}
     θ_r::FT
     "Hydraulic conductivity of wet snow (m/s)"
     Ksat::FT
-    "Critical threshold for snow cover fraction (m)"
-    fS_c::FT
     "Thermal conductivity of ice (W/m/K)"
     κ_ice::FT
-    "Timestep of the model"
-    Δt::Any
+    "Timestep of the model (s)"
+    Δt::FT
+    "Areal specific heat of ground interacting with snow (J/m^2/K)"
+    ρcD_g::FT
     "Clima-wide parameters"
     earth_param_set::PSE
 end
 
-include("./snow_parameterizations.jl")
+"""
+   SnowParameters{FT}(Δt;
+                      ρ_snow = FT(200),
+                      z_0m = FT(0.0024),
+                      z_0b = FT(0.00024),
+                      α_snow = FT(0.8),
+                      ϵ_snow = FT(0.99),
+                      θ_r = FT(0.08),
+                      Ksat = FT(1e-3),
+                      κ_ice = FT(2.21),
+                      ρcD_g = FT(3.553e5),
+                      earth_param_set::PSE) where {FT, PSE}
+
+An outer constructor for `SnowParameters` which supplies defaults for
+all arguments but `earth_param_set`.
+"""
+function SnowParameters{FT}(
+    Δt;
+    ρ_snow = FT(200),
+    z_0m = FT(0.0024),
+    z_0b = FT(0.00024),
+    α_snow = FT(0.8),
+    ϵ_snow = FT(0.99),
+    θ_r = FT(0.08),
+    Ksat = FT(1e-3),
+    κ_ice = FT(2.21),
+    ρcD_g = FT(3.553e5),
+    earth_param_set::PSE,
+) where {FT <: AbstractFloat, PSE}
+    return SnowParameters{FT, PSE}(
+        ρ_snow,
+        z_0m,
+        z_0b,
+        α_snow,
+        ϵ_snow,
+        θ_r,
+        Ksat,
+        κ_ice,
+        Δt,
+        ρcD_g,
+        earth_param_set,
+    )
+end
+
+Base.broadcastable(ps::SnowParameters) = tuple(ps)
+
+"""
+    struct SnowModel{
```

Comment:
I think this is ok for now, but yet another open question, should we give more specific names for the sake of modularity (we may have multiple snow models in the future)